jupyter-book/overfit/overfit_intro.md

```diff
@@ -0,0 +1,23 @@
+# Overfitting and underfitting
+
+This chapter gives an intuitive introduction to the very **fundamental concepts** of
+overfitting and underfitting in machine learning.
+
+Machine learning models can never make perfect predictions: the test error is
+never exactly zero. This failure comes from a **fundamental trade-off** between
+**modeling flexibility** and the **limited size of the training dataset**.
+
+The first presentation will define those problems and characterize how and why
+they arise.
+
+Then we will present a methodology to quantify those problems by **contrasting the
+train error with the test error** for various choice of the model family,
+model parameters. More importantly, we will emphasize the **impact of the size of
+the training set on this trade-off**.
+
+Finally we will relate overfitting and underfitting to the concepts of
+statistical variance and bias.
+
+```{tableofcontents}ß
```

Comment:
Naive question: is the "beta" at the end of this line desired?

I'm not quite sure what it does.

Reply:
hehe, no it's a typo